app/playground/page.tsx

```diff
@@ -0,0 +1,30 @@
+import { Metadata } from 'next';
+import PageClient from './page.client';
+
+export const metadata: Metadata = {
+  title: 'EDPN Playground',
+  description: 'Elite Dangerous Pilots Network',
+  icons: 'EDPN_logo.png',
```

Comment:
```suggestion
  icons: 'EDPN_logo_black.png',
```

This also needs updated in the FAQ page.tsx

Reply:
Will update 👍